frontend/src/components/program/PeriodSwitcher.vue

```diff
@@ -4,20 +4,28 @@
       <template #activator="{ on, attrs, value }">
         <v-btn
           text large
-          class="justify-start pl-0"
+          class="justify-start px-2"
           height="auto" block
           v-bind="attrs"
           v-on="on">
-          <h1 class="text-h6">
+          <h1 class="text-subtitle-1">
             {{ period().description }}
           </h1>
           <v-icon v-if="value" right>mdi-menu-up</v-icon>
           <v-icon v-else right>mdi-menu-down</v-icon>
         </v-btn>
       </template>
       <v-list>
-        <v-list-item v-for="item in period().camp().periods().items" :key="item._meta.self" :to="periodRoute(item)">
-          {{ item.description }}
+        <v-subheader>{{ $tc('components.camp.periodSwitcher.title') }}</v-subheader>
+        <v-list-item v-for="item in period().camp().periods().items" :key="item._meta.self"
+                     :to="periodRoute(item)"
+                     two-line>
+          <v-list-item-content>
+            <v-list-item-title>{{ item.description }}</v-list-item-title>
+            <v-list-item-subtitle>
+              {{ $date.utc(item.start).format($tc('global.datetime.dateShort')) }} - {{ $date.utc(item.end).format($tc('global.datetime.dateShort')) }}
```

Comment:
Alternatively, you could also use `rangeShort(item.start, item.end)` from `dateHelperUTCFormatted.js`

Reply:
I don't think this makes sense, as this includes the time. I've created a new dateRange helper.

Reply:
Ah yes, true. Mixed it up. Well solved.